worlds/metroidprime/Config.py

```diff
@@ -0,0 +1,365 @@
+        "Artifact of Wild": make_artifact_hint("Artifact of Wild"),
+        "Artifact of Lifegiver": make_artifact_hint("Artifact of Lifegiver"),
+        "Artifact of Warrior": make_artifact_hint("Artifact of Warrior"),
+    }
+
+
+def get_tweaks(world: "MetroidPrimeWorld") -> Dict[str, List[float]]:
+    color = color_options_to_value(world)
+    if color != HudColor.DEFAULT.value:
+        return {"hudColor": color}
+    else:
+        return {}
+
+
+def get_strg(world: "MetroidPrimeWorld") -> Dict[str, List[str]]:
+    strg = {**OBJECTIVE_STRG}
+    # Set objective text in temple security station
+    objective_text = f"Current Mission: Retrieve {world.options.required_artifacts}  Chozo Artifact{'s' if world.options.required_artifacts != 1 else ''}"
+    if world.options.final_bosses == 0:
+        objective_text += "\nDefeat Meta Ridley\nDefeat Metroid Prime"
+    elif world.options.final_bosses == 1:
+        objective_text += "\nDefeat Meta Ridley"
+    elif world.options.final_bosses == 2:
+        objective_text += "\nDefeat Metroid Prime"
+
+    strg[DEFAULT_OBJECTIVE_STRG_KEY][2] = objective_text
+
+    return strg
+
+
+def make_version_specific_changes(
+    config_json: Dict[str, Any], version: str
+) -> Dict[str, Any]:
+    if DEFAULT_OBJECTIVE_STRG_KEY in config_json["strg"] and version not in [
+        "0-00",
+        "0-01",
+        "0-02",
+        "kor",
+    ]:
+        value = config_json["strg"][DEFAULT_OBJECTIVE_STRG_KEY]
+        del config_json["strg"][DEFAULT_OBJECTIVE_STRG_KEY]
+        config_json["strg"][get_objective_strg_key(version)] = value
+
+    return config_json
+
+
+def make_config(world: "MetroidPrimeWorld") -> Dict[str, Any]:
+    options: MetroidPrimeOptions = world.options
+    config: Dict[str, Any] = {
+        "$schema": "https://randovania.org/randomprime/randomprime.schema.json",
+        "inputIso": "prime.iso",
+        "outputIso": "prime_out.iso",
+        "forceVanillaLayout": False,
+        "strg": get_strg(world),
+        "preferences": {
+            "forceFusion": bool(options.fusion_suit.value),
+            "cacheDir": "cache",
+            "qolGeneral": True,
+            "qolGameBreaking": True,
+            "qolCosmetic": True,
+            "qolCutscenes": "Skippable",
+            "qolPickupScans": True,
+            "mapDefaultState": "Always",
+            "artifactHintBehavior": "All",
+            "skipSplashScreens": bool(os.environ.get("DEBUG", False)),
+            "quickplay": bool(os.environ.get("DEBUG", False)),
+            "quickpatch": bool(os.environ.get("DEBUG", False)),
+            "quiet": bool(os.environ.get("DEBUG", False)),
+            "suitColors": {
+                "gravityDeg": world.options.gravity_suit_color.value or 0,
+                "phazonDeg": world.options.phazon_suit_color.value or 0,
+                "powerDeg": world.options.power_suit_color.value or 0,
+                "variaDeg": world.options.varia_suit_color.value or 0,

```

Comment:
This options are all `Range` options, so I don't think there is any way they should end up with a `.value` of `None`.

```suggestion
                "gravityDeg": world.options.gravity_suit_color.value,
                "phazonDeg": world.options.phazon_suit_color.value,
                "powerDeg": world.options.power_suit_color.value,
                "variaDeg": world.options.varia_suit_color.value,
```